cmd/tailscale/cli/cli.go

```diff
@@ -207,7 +207,7 @@ change in the future.
 			debugCmd,
 			driveCmd,
 			idTokenCmd,
-		},
+		}, maybeAdvertiseCmd()...),
```

Comment:
this feels a little messy but I guess we don't currently have any completely conditional subcommands, so I guess it makes sense for now